code/modules/pda/cart_apps.dm

```diff
@@ -148,7 +148,7 @@
 
 	. = TRUE
 
-	if(!pda.silent)
+	if(!pda?.silent)
```

Comment:
```suggestion
	if(pda && !pda.silent)
```
Else it would play a sound in nullspace

Reply:
I really hate double negatives sometimes. (If this even is one)